src/main/resources/locales/en/translation/type-of-claim.json

```diff
@@ -36,7 +36,8 @@
       "required": "Please select at least one of the options below to choose your type of claim."
     },
     "otherClaim": {
-      "required": "Please describe the type of claim"
+      "required": "Please describe the type of claim",
+      "tooLong": "Description must be 2000 characters or less"
```

Comment:
might be worth creating an additional ticket to obtain a Welsh translation for this

Reply:
https://tools.hmcts.net/jira/browse/RET-3205